src/main/java/ch/njol/skript/effects/EffItemDespawn.java

```diff
@@ -0,0 +1,61 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Item Despawn")
+@Description("Prevent a dropped item from naturally despawning through Minecraft's timer.")
+@Examples({
+	"prevent all dropped items from naturally despawning",
+	"allow all dropped items to naturally despawn"
+})
+@Since("INSERT VERSION")
+public class EffItemDespawn extends Effect {
+
+	static {
+		Skript.registerEffect(EffItemDespawn.class,
+			"(prevent|disallow) %itementities% from (naturally despawning|despawning naturally)",
+			"allow %itementities% to (naturally despawn|despawn naturally)");
+	}
```

Comment:
this is optional, but I think I'd like this
```suggestion
		Skript.registerEffect(EffItemDespawn.class,
			"(prevent|disallow) %itementities% from (naturally despawning|despawning naturally)",
			"allow natural despawning of %itementities%",
			"allow %itementities% to (naturally despawn|despawn naturally)");
	}
```